tests/scalers/gcp/gcp_cloud_tasks_workload_identity/gcp_cloud_tasks_workload_identity_test.go

```diff
@@ -0,0 +1,258 @@
+//go:build e2e
+// +build e2e
+
+package gcp_cloud_tasks_workload_identity_test
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/joho/godotenv"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/client-go/kubernetes"
+
+	. "github.com/kedacore/keda/v2/tests/helper"
+)
+
+// Load environment variables from .env file
+var _ = godotenv.Load("../../.env")
+
+var now = time.Now().UnixNano()
+
+const (
+	testName = "gcp-cloud-tasks-test"
```

Comment:
This should be different to avoid collisions with the other tests. As this is used as base for all the resources, both test in this PR are deploying the same 😄

Reply:
```suggestion
	testName = "gcp-cloud-tasks-workload-identity-test"
```